src/api/app/models/user.rb

```diff
@@ -753,8 +753,8 @@ def declined_requests(search = nil)
 
   # list incoming requests involving this user
   def incoming_requests(search = nil)
-    result = BsRequest.with_involved_projects(involved_projects.pluck(:id)).or(
-      BsRequest.with_involved_packages(involved_packages.pluck(:id))
+    result = BsRequest.where(id: BsRequestAction.where(target_project_id: involved_projects.pluck(:id)).select(:bs_request_id)).or(
+      BsRequest.where(id: BsRequestAction.where(target_package_id: involved_packages.pluck(:id)).select(:bs_request_id))
```

Comment:
Can you move this code back into scopes like it was done before? Before is much more readable then now.. (Thats what I meant by my comment in the other file "Why get rid of these scopes?")